docs/api/12_customer_usage/custome-usage.mdx

```diff
@@ -0,0 +1,132 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Get customer current usage
+
+## Route
+
+```curl title="GET"
+/api/v1/customers/:customer_id/current_usage
+```
+
+## Usage
+
+<Tabs groupId="prog-language">
+  <TabItem value="curl" label="Curl" default>
+
+  ```bash
+  LAGO_URL="https://api.getlago.com"
+  API_KEY="__YOUR_API_KEY__"
+  CUSTOMER_ID="__CUSTOMER_ID__"
+
+  curl --location --request GET "$LAGO_URL/api/v1/customers/$CUSTOMER_ID/current_usage"
+    --header "Authorization: Bearer $API_KEY" \
+    --header 'Content-Type: application/json'
+  ```
+
+  </TabItem>
+
+  <TabItem value="python" label="Python">
+
+  ```python
+  from lago_python_client import Client
+
+  client = Client(api_key='__YOUR_API_KEY__')
+
+  customer_usage = client.customers().current_usage(customer_id='customer_id')
+  ```
+
+  </TabItem>
+
+  <TabItem value="javascript" label="Node.js">
+
+  ```javascript
+  import Client from 'lago-nodejs-client'
+
+  let client = new Client('__API_KEY__')
+
+  let customerUsage = await client.customerCurrentUsage('customer_id')
+  ```
+
+  </TabItem>
+
+  <TabItem value="ruby" label="Ruby">
+
+  ```ruby
+  require 'lago-ruby-client'
+
+  client = Lago::Api::Client.new({api_key: '__YOUR_API_KEY__'})
+  customer_usage = client.customer.current_usage('')
```

Comment:
Should it be `('')` or `('customer_id')` at the end?

Reply:
`('customer_id')`, good catch!